graphql/schema.graphql

```diff
@@ -902,6 +903,31 @@ type JiraStatus {
   name: String!
 }
 
+type Annotation {
+  taskId: String!
+  taskExecution: Int!
+  Note: Note
+  issues: [IssueLink]
+  suspectedIssues: [IssueLink]
+}
+
+type Note {
+  message: String!
+  source: Source!
+}
+
+type IssueLink {
```

Comment:
Should AnnotationIssue (the input  type) be called IssueLinkInput to match this? (I'm not sure if we're aiming for consistency with this or not)

Reply:
good idea